src/pages/Home.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { } from 'react';
+
+function Home() {
```

Comment:
Please remove `Page` from the names of the other pages and remove the other `HomePage.jsx` file (use `git mv` and `git rm` to do renames/removes)